### ISSUE TITLE
Spelling: collapsable should be collapsible

### ISSUE THREAD

**Issue by abelbraaksma** [Microsoft/visualfsharp#5247]
In the Advanced tab: 
![image](https://user-images.githubusercontent.com/16015770/42051801-4b4d395c-7b0d-11e8-9d3e-775fccac102e.png)

The preferred spelling is _collapsible_, not _collapsable_ (and Microsoft's spelling check agrees, as does the open source spelling check of Firefox). See [[1]](https://www.drupal.org/project/views_pdf/issues/1270346), [[2]](https://www.merriam-webster.com/dictionary/collapsible), [[3]](https://github.com/react-bootstrap/react-bootstrap/issues/425).

**Comment by dsyme** [comment]
@abelbraaksma Can you submit a PR please? thanks

**Comment by abelbraaksma** [comment]
@dsyme, done (I think, I am not too used to doing PR's, so if I forgot something for this trivial request, please let me know ;).
